### ISSUE TITLE
Outputs on Run JavaScript node?

### ISSUE THREAD

**Issue by surma** [breadboard-ai/breadboard#3628]
When I create a new item on the input schema of the Run JavaScript node, it shows up as a separate input.

When I create a new item on the output schema of the Run JavaScript node, nothing happens.

Is that intentional? I find myself getting confused as to what the return value of my JS function should be. Should it be an object with one property per output schema item? What actually goes onto the wire also seems to depend on whether I select "Context" or one of the other options for its type.

**Comment by dglazkov** [comment]
Yeah, that's the sins of the past haunting us. Check "Raw" checkbox and the schema outputs will pop up. The "Run Module" can't come soon enough.

**Comment by dglazkov** [comment]
Similar #3527
